packages/core/test/logger.test.ts

```diff
@@ -7,141 +7,146 @@ const TEST_WARN_MESSAGE = 'Test warn message'
 const TEST_ERROR_MESSAGE = 'Test error message'
 const TEST_DEBUG_MESSAGE = 'Test debug message'
 
-describe('core/logger.ts', () => {
+describe('core/logger.ts DefaultLogger', () => {
```

Comment:
Also just renames in this file `Mock` -> `Spy` for clarity. This is where I would have started the `emitLogs` testing but I backed off when I realized I was going too far down the rabbit hole.